source/change-streams/tests/unified/change-streams-showExpandedEvents.yml

```diff
@@ -0,0 +1,299 @@
+description: "change-streams-showExpandedEvents"
+schemaVersion: "1.7"
+runOnRequirements:
+  - minServerVersion: "6.0.0"
+    topologies: [ replicaset, sharded-replicaset, sharded ]
+createEntities:
```

Comment:
There are a lot of entities here. I don't see usages of `globalClient`, `database1`, `collection1`, `globalDatabase0`, `globalCollection0`, `globalDatabase1`, `globalCollection1`, `globalDb1Collection0` or `globalDb0Collection1` in the tests, but I might be missing something. Do we need those entities?

Reply:
Those are leftover from my initial tests trying to write tests for `refineCollectionShardKey` and `reshardCollection` - I must have forgotten to remove them.  I remove any of the unused entities.